Dockerfile

```diff
@@ -4,9 +4,17 @@ FROM google/dart-runtime-base:1.23.0
 # docker image diff small.
```

Comment:
Maybe change the `FROM ...` to upgrade to the latest stable version, namely 1.24.0.